driver-core/src/main/com/mongodb/client/model/expressions/ArrayExpression.java

```diff
@@ -50,25 +50,30 @@
      */
     <R extends Expression> ArrayExpression<R> map(Function<? super T, ? extends R> in);
 
+    ArrayExpression<T> sort();
+
     IntegerExpression size();
 
-    BooleanExpression any(Function<T, BooleanExpression> map);
+    BooleanExpression any(Function<T, BooleanExpression> mapper);
+
+    BooleanExpression all(Function<T, BooleanExpression> mapper);
 
-    BooleanExpression all(Function<T, BooleanExpression> map);
+    NumberExpression sum(Function<T, NumberExpression> mapper);
 
-    NumberExpression sum(Function<T, NumberExpression> map);
+    NumberExpression multiply(Function<T, NumberExpression> mapper);
 
-    NumberExpression multiply(Function<T, NumberExpression> map);
+    <R extends Expression> R max(Function<T, R> mapper, R orElse);
 
-    NumberExpression max(Function<T, NumberExpression> map, NumberExpression orElse);
+    <R extends Expression> R min(Function<T, R> mapper, R orElse);
```

Comment:
Similar usages of `orElse` were changed to `other` in https://github.com/mongodb/mongo-java-driver/pull/1052. We should use `other` in `min`/`max`.

Reply:
I am second-guessing that change, but I carried it through to this.